### ISSUE TITLE
 Problem with routes

### ISSUE THREAD

**Issue by osek666** [imindeu/play-cxf#8]
I have Play Framework 2.4.3 (java) and implemented my web service according to your instructions here http://www.imind.eu/web/2013/11/07/developing-soap-services-using-play-framework-2-2-x/
Unfortunately I get compilation error:
```
(...)/conf/routes:31:  type CxfController is not a member of package org.apache.cxf.transport.play
```
It is related with routes. I have no idea how to fix it. Could you help me?

**Comment by kustra** [comment]
Hello,
please check if you have play-cxf_play24 as the dependency name. If this
doesn't help, please post your build.sbt here.
L
2015.09.25. 9:24 ezt írta ("osek666" <notifications@github.com>):

**Comment by osek666** [comment]
Hi, 

my build.sbt:

```
name := "project_name"

version := "1.0-SNAPSHOT"

lazy val root = (project in file(".")).enablePlugins(PlayJava)

scalaVersion := "2.11.7"

resolvers ++= Seq(
  "Nexus" at "http://(...)/content/repositories/releases"
)

libraryDependencies ++= Seq(
  "org.springframework" % "spring-context" % "4.1.6.RELEASE",
  "eu.imind.play" % "play-cxf_play24_2.11" % "1.2.0" exclude("org.springframework", "spring-context"),
  "org.apache.cxf" % "cxf-rt-bindings-soap" % "3.1.2",
  "org.apache.cxf" % "cxf-rt-frontend-jaxws" % "3.1.2",
  (...)
)

// Play provides two styles of routers, one expects its actions to be injected, the
// other, legacy style, accesses its actions statically.
routesGenerator := InjectedRoutesGenerator

```

**Comment by kustra** [comment]
This seems ok to me. What Java version are you using? Can you run the
play-cxf-hello sample from the repository's play-2.4.x branch?
L
2015.09.25. 9:48 ezt írta ("osek666" <notifications@github.com>):

**Comment by osek666** [comment]
I'm using Java 8. It is weird because sample works.

**Comment by kustra** [comment]
Then please put together a minimal sample that produces the issue and send
it to me. Or if you find it according to the sample, let me know!
L
2015.09.25. 10:36 ezt írta ("osek666" <notifications@github.com>):

**Comment by kustra** [comment]
Fixed with version 1.2.1.
